utils/network/utils/http-error-handler.ts

```diff
@@ -18,6 +19,9 @@ export const httpErrorHandler = (
   message: string
 ) => {
   switch (status) {
+    case 432: {
+      throw new AgentOverRateLimit(statusText || 'Agent over rate limit');
+    }
```

Comment:
Does not really make sense here ? No client would return a 432

Reply:
Actually we have to do this, otherwise, the 432 becomes a 500 :
<img width="518" alt="image" src="https://github.com/user-attachments/assets/2f9357fa-6616-4577-b0ee-c8bd511d6d4a" />
<img width="632" alt="image" src="https://github.com/user-attachments/assets/2941118d-0660-43cf-b59b-318a8ca00e07" />

Reply:
(because of how httpErrorHandler works)